include/deal.II/lac/petsc_ts.h

```diff
@@ -438,6 +455,26 @@ namespace PETScWrappers
     unsigned int
     solve(VectorType &y, AMatrixType &A, PMatrixType &P);
 
+    /**
```

Comment:
Functions that should not be called by users should be in the private interface. Would you mind moving this and the one below in the private interface?

Reply:
They cannot be `private`. These are called within a `TS` callback that cannot capture `[&]`. See e.g. https://github.com/dealii/dealii/pull/15785/files#diff-e7b3d4a53e4de44768fd65ddf265235d71afd16048278eeaa789f4d85a14b6a7R507

Reply:
Ah. We already discussed this, didn't we? :D I completely forgot. Thanks for your patience... :)